bot/exts/__init__.py

```diff
@@ -1,9 +1,8 @@
 import logging
 import pkgutil
-from pathlib import Path
 from typing import Iterator
 
-__all__ = ("get_package_names", "walk_extensions")
+__all__ = ("get_package_names",)
```

Comment:
Perhaps can you also move this `get_package_names` to an utile file?

Reply:
Not quite sure how you got that version. `"walk_extensions"` was removed from `exts/__init__.py` in 921f180.

Reply:
Yes, it would be nice if you can move over the other function too 🙂

Reply:
When moving it to `bot/utils`, the following error is generated:
```
Extension 'bot.exts.evergreen.branding' raised an error: NameError: name '__path__' is not defined
```
I think it would be fine to keep it in `__init__.py`. The code is minimal.

Reply:
Okay, fixing this would be out of scope for this PR.